### DISCUSSION TITLE
Title Auto Gen will Sometimes Include Markdown

### DISCUSSION THREAD

**Discussion by hchasens** [open-webui/open-webui#2420] [General]
# Bug Report

## Description

**Bug Summary:**
Title generated will sometimes include markdown (e.g. "**Malicious Code Detection**")

**Steps to Reproduce:**
This is probably RNG so it'd be difficult to recreate. 

**Expected Behavior:**
Markdown not appearing in the title as it fails to parse. 

**Actual Behavior:**
Markdown will sometimes appear in the title. 

## Environment

- **Open WebUI Version:** [v0.1.125]
- **Ollama (if applicable):** [0.1.38]

- **Operating System:** [e.g., Arch Linux]
- **Browser (if applicable):** [e.g., Vivaldi]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

**Screenshots (if applicable):**
![issue](https://github.com/open-webui/open-webui/assets/16534666/84c7b792-b1c5-4a3a-89ce-f1c4b08922b0)

## Additional Information

Can be fixed by changing the title gen prompt to:
> Create a concise, 3-5 word phrase as a header for the following query, strictly adhering to the 3-5 word limit and avoiding the use of the word 'title' and don't use markdown: {{prompt}}

**Comment by cheahjs** [comment]
see: https://github.com/open-webui/open-webui/discussions/1692
